source/index.ts

```diff
@@ -1120,7 +1124,7 @@ const methodTypeMap = {
 	isWhitespaceString: 'whitespace string',
 } as const;
 
-function keysOf<T extends Record<PropertyKey, unknown>>(value: T): Array<keyof T> {
+export function keysOf<T extends Record<PropertyKey, unknown>>(value: T): Array<keyof T> {
```

Comment:
This should not be exposed to users. Would be better to move it to a `utilities` file and import it into both here and tests.

Reply:
Done!